pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -715,6 +716,20 @@ func GetOpenshiftBootstrapClusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule("get", "put", "update", "delete").URLs(templateapi.ServiceBrokerRoot + "/*").RuleOrDie(),
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "resource-metrics-view",
+				Labels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-view": "true", "rbac.authorization.k8s.io/aggregate-to-admin": "true"}},
+			Rules: []rbacv1.PolicyRule{
+				rbacv1helpers.NewRule(read...).Groups(metrics.GroupName).Resources("pods").RuleOrDie(),
+			},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "resource-metrics-cluster-view", Labels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-cluster-reader": "true"}},
+			Rules: []rbacv1.PolicyRule{
+				rbacv1helpers.NewRule(read...).Groups(metrics.GroupName).Resources("nodes").RuleOrDie(),
+				rbacv1helpers.NewRule(read...).Groups(metrics.GroupName).Resources("pods").RuleOrDie(),
```

Comment:
This rule should not be here, `view` already aggregates into `cluster-reader`.

Reply:
Please provide me with a one sentence description of the nodes resource so I can understand if it makes sense to aggregate into cluster reader.

Reply:
nodes.metrics.k8s.io allows you to view CPU and memory usage for nodes.